stubs/ttkthemes/ttkthemes/themed_tk.pyi

```diff
@@ -21,10 +21,56 @@ class ThemedTk(tkinter.Tk, ThemedWidget):
         background: bool | None = ...,  # old alias for themebg
         gif_override: bool = ...,
     ) -> None: ...
-    def set_theme(self, theme_name, toplevel: bool | None = None, themebg: bool | None = None) -> None: ...
+    def set_theme(self, theme_name: str, toplevel: bool | None = None, themebg: bool | None = None) -> None: ...
     # TODO: currently no good way to say "use the same big list of kwargs as parent class but also add these"
```

Comment:
Maybe this comment should be replaced with something like "Please keep this in sync with tkinter.Tk"? You should also add corresponding comment to stubs of `tkinter.Tk` in stdlib, so that if one is changed, the other is changed as well. Otherwise these may get out of sync over time (although tkinter stubs are quite stable now).